source/tasks/Utils/environment.ts

```diff
@@ -200,7 +200,7 @@ export const getVcsTypeFromProvider = (buildRepositoryProvider: string) => {
 export const getBuildBranch = async (client: vsts.WebApi) => {
     const environment = getVstsEnvironmentVariables();
     const api = await client.getBuildApi();
-    return (await api.getBuild(environment.buildId, environment.projectName)).sourceBranch;
+    return (await api.getBuild(environment.projectName, environment.buildId)).sourceBranch;
```

Comment:
So have we had this backwards and not noticed? Or did the API change?

Reply:
the api changed

Reply:
👍